Lib/Filter/CssUrlRewrite.php

```diff
@@ -0,0 +1,85 @@
+<?php
+App::uses('AssetFilter', 'AssetCompress.Lib');
```

Comment:
Your pull request is targetting the `master` branch, which is a 3.x compatible version of AssetCompress, but this pull request using 2.x conventions. Are you intending to update the plugin for 2.x or for 3.x?